posthog/migrations/0710_add_salesforce_tables.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 4.2.18 on 2025-04-15 10:15
+
+from django.db import connection, migrations
+
+
+def insert_salesforce_events_tasks(apps, schema_editor):
+    with connection.cursor() as cursor:
+        cursor.execute("SELECT id, team_id FROM posthog_externaldatasource where source_type = 'Salesforce'")
+        salesforce_sources = cursor.fetchall()
+
+    ExternalDataSchema = apps.get_model("posthog", "ExternalDataSchema")
+    for source in salesforce_sources:
```

Comment:
Seems a bit wasteful to do so many individual inserts here, couldn't we use [the `bulk_create` api](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#bulk-create)?

Reply:
Its a total of 62 inserts in the US which felt like an appropriate amount to not use the bulk API